XamlAnimatedGif/UriLoader.cs

```diff
@@ -13,26 +13,26 @@ namespace XamlAnimatedGif
 {
     internal class UriLoader
     {
-        public static Task<Stream> GetStreamFromUriAsync(Uri uri, IProgress<int> progress)
+        public static Task<Stream> GetStreamFromUriAsync(Uri uri, string tempPath, IProgress<int> progress)
```

Comment:
No need for the extra parameter. As per my earlier comment, this should be a static field on the UriLoader class, set by the `AnimationBehavior.SetDownloadCacheLocation` method.